platform/darwin/src/MGLSymbolStyleLayer.mm

```diff
@@ -84,6 +84,11 @@
         { MGLTextPitchAlignmentAuto, "auto" },
     });
 
+    MBGL_DEFINE_ENUM(MGLTextPlacementMode, {
```

Comment:
This also has to add a round trip conversion to `NSValue`. Take a look at https://github.com/mapbox/mapbox-gl-native/blob/master/platform/darwin/src/MGLSymbolStyleLayer.mm#L1490

Reply:
done